native/src/main/java/io/ballerina/lib/ldap/Client.java

```diff
@@ -121,13 +129,13 @@ public static Object modify(Environment env, BObject ldapClient,
         return null;
     }
 
-    public static Object modifyDN(Environment env, BObject ldapClient, BString currentDN,
-                                  BString newRDN, boolean deleteOldRDN) {
+    public static Object modifyDn(Environment env, BObject ldapClient, BString currentDn,
+                                  BString newRdn, boolean deleteOldRdn) {
```

Comment:
Is there any difference between what we have defined as `dn`s and `rdn`s?

Reply:
Yeah, DN contains the absolute path for an entry while RDN (Relative Distinguished Name) contains only last component of the path

Reply:
Got it